src/helpers/get-int-config.ts

```diff
@@ -0,0 +1,21 @@
+import { InvalidConfigException } from '@ukef/config/invalid-config.exception';
+
+// This helper function is used to get integer from configuration.
+export const getIntConfig = (environmentVariable: string, defaultValue?: number): number | undefined => {
```

Comment:
Does this just return number?

Reply:
yes, number, but specifically just integer.

Reply:
We could create type for integer, but not sure if this is required.

Reply:
Sorry - I mean why are we doing `number | underfined`

Reply:
Good catch, it is from older version of the function. I removed `undefined` from Estore and MDM APIs.